src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -241,11 +245,59 @@ const DismissButton = styled(Button)`
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 `;
+const ExampleEventCause = styled(Panel)`
+  overflow: hidden;
+  pointer-events: none;
+`;
+
+const ExampleRibbon = styled('div')`
+  position: absolute;
+  right: 0px;
+
+  &:after {
+    display: block;
+    content: 'Example';
+    position: absolute;
+    top: 16px;
+    right: -24px;
+    text-transform: uppercase;
+    background: #e46187;
+    padding: 4px 26px;
+    line-height: 11px;
+    font-size: 11px;
+    color: ${p => p.theme.white};
+    transform: rotate(45deg);
+  }
```

Comment:
Can you add `pointer-events: none` so that we can click through it?

Reply:
I added it to `ExampleEventCause` so you can't click the commit hash

Reply:
ah cool